testsuite/MDAnalysisTests/utils/test_imports.py

```diff
@@ -22,36 +22,41 @@
 from __future__ import absolute_import
 
 import os
+
 import MDAnalysisTests
+import pytest
+
+"""Relative imports are banned in unit testing modules (Issue #189), so run tests to enforce this policy."""
+
+path_to_testing_modules = MDAnalysisTests.__path__[0]
+# Exclusion path relative to MDAnalysisTests
+exclusions = ['/plugins', '/data']
```

Comment:
Yeah I'd have preferred this too, it makes it obvious where things have come from